docs/README.md

```diff
@@ -3,7 +3,7 @@
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-# Kubernetes Documentation: releases.k8s.io/v1.3.0-beta.0
+# Kubernetes Documentation: releases.k8s.io/release-1.3
```

Comment:
Supposed to be a branch.  See also https://github.com/kubernetes/release/pull/14